OsmAnd/src/net/osmand/plus/search/QuickSearchDialogFragment.java

```diff
@@ -1329,6 +1329,7 @@ private void reloadCitiesInternal() {
 		if (SearchUICore.isDebugMode()) {
 			LOG.info("UI >> Start loading nearest cities");
 		}
+		updateCitiesItems();
```

Comment:
this could lead to concurrentmodificationexception as the search is not finished and running when it starts updating... but probably not if it's sync